lib/galaxy/webapps/galaxy/api/datasets.py

```diff
@@ -194,18 +194,26 @@ def extra_files(
     ):
         return self.service.extra_files(trans, history_content_id)
 
+    @router.get(
+        "/api/datasets/{history_content_id}/display",
+        summary="Displays (preview) or downloads dataset content.",
+        response_class=StreamingResponse,
+    )
```

Comment:
It should work, but then this route will appear under `histories` or the other one under `datasets` because I can not set a different tag for the alias right?

Reply:
If the tag is not important, I'm happy to use the new alias parameter :)